bin/evidence_string_generation.py

```diff
@@ -1,24 +1,16 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
 
 import sys
-
 from eva_cttv_pipeline.evidence_string_generation import utilities, clinvar_to_evidence_strings
 
 
 def main():
     parser = utilities.ArgParser(sys.argv)
-
-    utilities.check_for_local_schema()
-
     utilities.check_dir_exists_create(parser.out)
-
-    clinvar_to_evidence_strings.launch_pipeline(parser.out,
-                                                allowed_clinical_significance=parser.clinical_significance,
-                                                efo_mapping_file=parser.efo_mapping_file,
-                                                snp_2_gene_file=parser.snp_2_gene_file,
-                                                json_file=parser.json_file)
-
-    print('>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> Finished <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<')
```

Comment:
Why take this print statement out?

Reply:
I feel that the unconditional print statement at the very end of the script is redundant because script already returns 0 on successful exit, and bsub will report the job as "Successfully completed" anyway.